### PR TITLE
Added signup page

### LINKED ISSUE #14: Added Signup page
Added signup page
Added responsiveness

### PR DESCRIPTION
**Pull request by Raghav888**
1. Added signup page
2.  Added responsiveness

### CODE REVIEW

File: src/components/products/Product.js [hunk 1]
```diff
@@ -22,12 +23,10 @@ export const Products = () => {
             return (
               <div className="mantra-vertical-card card-holder">
                 <div class="mantra-icon mantra-close-icon">
-                  <i class="fas fa-heart-circle"></i>
+                  <i class="fas fa-heart-circle icon-size"></i>
```
Comment (Bhakti199) [inline src/components/products/Product.js:26]: you may use react-icons or fontawesome's icons for react (for `className`)

File: src/pages/signup/Signup.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./signup.css";
+
+import { Navbar } from "../../components/shared/navbar/Navbar";
+import { Signupform } from "../../components/signupform/Signupform";
```
Comment (Bhakti199) [inline src/pages/signup/Signup.js:5]: you may use common index file to export all your components from parent folder, it will easy to access

**Review by Bhakti199 — APPROVED** [review]
- Overall code is nicely written and structured properly.
- proper naming conventions
- like that defining of small components for easy access.
Suggested some changes, this PR is good to merge

File: src/components/signupform/Signupform.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+              id="name"
+              className="text-box mantra-textbox-classic mantra-highlight-box"
+              type="text"
+              placeholder=" xyz "
+              required
+            />
+          </div>
+          <div className="signup-input">
+            <label for="last" className="item-font">
+              Last Name:{" "}
```
Comment (rkapoor10) [inline src/components/signupform/Signupform.js:28]: Remove `{" "}`  if not added by you

File: src/components/signupform/Signupform.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+              type="password"
+              placeholder="  ********** "
+              required
+            />
+          </div>
+
+          <div className="signup-input">
+            <input type="checkbox" id="remember-me" />
+            <label for="remember-me" className="item-font">
+              {" "}
```
Comment (rkapoor10) [inline src/components/signupform/Signupform.js:87]: same as above

File: src/components/signupform/Signupform.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+              className="text-box mantra-textbox-classic mantra-highlight-box"
+              type="password"
+              placeholder="  ********** "
+              required
+            />
+          </div>
+
+          <div className="signup-input">
+            <input type="checkbox" id="remember-me" />
+            <label for="remember-me" className="item-font">
```
Comment (rkapoor10) [inline src/components/signupform/Signupform.js:86]: also instead of `for` use `htmlFor` here

**Review by rkapoor10 — APPROVED** [review]
The code is well structured.  Go through the suggestions once and the code is set to merge.

**Pull request merged**